test/parallel/test-tls-psk-circuit.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const tls = require('tls');
+
+const CIPHERS = 'PSK+HIGH';
+const USERS = {
+  UserA: 'd731ef57be09e5204f0b205b60627028',
```

Comment:
It wasn't clear in the API docs - is there any limit (upper or lower) on the size of the keys? Do they have to be same size as the underlying cipher? Is there an optimal size for security?

Reply:
I'll document the limits mentioned in https://github.com/nodejs/node/pull/23188#discussion_r229051040.
Though I don't know of any numbers for optimal security.